web/frontend/angular/src/app/dashboard-pages/notifications/notifications.component.ts

```diff
@@ -1,12 +1,58 @@
 import {Component, OnInit} from "@angular/core"
+import {ApisService} from "src/app/apis/apis.service"
+import * as $ from "jquery"
```

Comment:
As I commented in #63, you never installed jQuery and you probably shouldn't be using it anyway.

Reply:
@alinaah You did not address this comment.